packages/quick_actions/quick_actions/android/src/main/java/io/flutter/plugins/quickactions/MethodCallHandlerImpl.java

```diff
@@ -49,7 +52,18 @@ public void onMethodCall(MethodCall call, MethodChannel.Result result) {
       case "setShortcutItems":
         List<Map<String, String>> serializedShortcuts = call.arguments();
         List<ShortcutInfo> shortcuts = deserializeShortcuts(serializedShortcuts);
-        shortcutManager.setDynamicShortcuts(shortcuts);
+        ThreadPoolExecutor executor =
```

Comment:
**Note:** `setDynamicShortcuts(...)` call moved to worker thread to fix CI failure.

Reply:
could this use the new pattern for background platform channels?

Reply:
I talked to @camsim99 and we can't use Background Platform Channels unless we do a breaking change to the platform_interface (since it would require splitting up the channels between a platform thread handler channel and a background thread handler channel).  I'm going to file a bug for implementing TaskQueues for methods inside of MethodChannels.  @stuartmorgan this is related to the issue we were talking about for moving channels out of the platform_interface.

Reply:
I filed an issue for methodchannel method specific taskqueues: https://github.com/flutter/flutter/issues/93828

I think it's better just to remove platform channels from the platform_interface though.